docs/tempo/website/operations/backend_search.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Backend Search
+weight: 9
+---
+
```

Comment:
Should we add one of those yellow banners to the top of this page to remind people this feature is experimental?

Reply:
good call. banner added.